musicbot/bot.py

```diff
@@ -1311,8 +1311,11 @@ def _gen_embed(self):
                         return Response(self.str.get('cmd-play-spotify-album-queued', "Enqueued `{0}` with **{1}** songs.").format(res['name'], len(res['tracks']['items'])))
                     elif 'playlist' in parts:
                         res = await self.spotify.get_playlist(parts[-3], parts[-1])
+                        while int(res["tracks"]["total"]) > len(res['tracks']['items']):
+                            resp = await self.spotify.get_playlist(parts[-3], parts[-1], offset=len(res['tracks']['items']))
+                            res['tracks']['items'].extend(resp['tracks']['items'])
```

Comment:
What is the purpose of these added lines?

Reply:
Spotify limits each request to 100 tracks. if the playlist has more than 100 tracks, it would add only the 100 first ones. which is what those lines fix.